packages/application-shell/src/components/application-shell/application-shell.mod.css

```diff
@@ -3,6 +3,10 @@
   --menuWidth: 200px;
 }
 
+:global(#app) {
+  height: 100%;
+}
```

Comment:
Was this something we removed before?

Reply:
No I think because I changed the layout to use css-grid (previously was flexbox) and this was missing for such pages.